sympy/integrals/tests/test_integrals.py

```diff
@@ -2098,6 +2098,17 @@ def test_issue_20782():
     assert integrate(f, (x, -oo, 1)) == 1
     assert integrate(-f, (x, -oo, 1)) == -1
 
+def test_issue_20781():
+    x_d = symbols('x_d')
+    x = symbols('x')
```

Comment:
These symbols are already defined above.

https://github.com/sympy/sympy/blob/6f3b69642b9f50db1434d34128f29d1628cae810/sympy/integrals/tests/test_integrals.py#L47-L47

Reply:
yes they are not needed. removing